### PR TITLE
#8151 fix measure tool window

### PR DESCRIPTION
**Pull request by barema4**
## Description
This PR allows the measure tool window to display on top of the layertree pane when dragging it over the surface

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

Measure tool window goes below layertree pane
#8151

**What is the new behavior?**

 Measure tool window displays  on top of  layertree pane

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: build/buildConfig.js [hunk 1]
```diff
@@ -1,340 +1,340 @@
-const assign = require('object-assign');
```
Comment (MV88) [inline build/buildConfig.js:1]: changes to this file should not be committed unless meaningful

File: build/webpack.config.js [hunk 2]
```diff
@@ -1,26 +1,73 @@
 const path = require("path");
 
-const themeEntries = require('./themes.js').themeEntries;
```
Comment (MV88) [inline build/webpack.config.js:3]: do you have some formatter that changes the syntax?
please disable it for this project and use eslint configuration present in package.json

also do not commit your local changes here

File: web/client/configs/localConfig.json [hunk 3]
```diff
@@ -41,12 +41,8 @@
       }, {
         "urlPattern": ".*rest/config.*",
         "method": "bearer"
-      },
-      {
-        "urlPattern": "http(s)?\\:\\/\\/gs-stable\\.geo-solutions\\.it(\\:443|\\:80)?\\/geoserver/.*",
-        "authkeyParamName": "authkey",
-        "method": "authkey"
-    }],
+      }
```
Comment (MV88) [inline web/client/configs/localConfig.json:44]: this must not be commited, eventually you can keep it only locally

**Review by MV88 — CHANGES_REQUESTED** [review]

File: build/webpack.config.js (hunk 2 above)
Comment (barema4) [reply build/webpack.config.js:3]: Thanks for the feedback @MV88 . Let me work on them.

File: web/client/configs/localConfig.json (hunk 3 above)
Comment (barema4) [reply web/client/configs/localConfig.json]: Fixed it

File: build/buildConfig.js (hunk 1 above)
Comment (barema4) [reply build/buildConfig.js:1]: Done

**Review by MV88 — APPROVED** [review]

**Comment by MV88** [comment]
@ElenaGallo please test in DEV

**Comment by tdipisa** [comment]
@ElenaGallo let us know if we can backport to 2022.01.xx.

**Comment by ElenaGallo** [comment]
Test passed, @barema4 please backport to 2022.01.xx. Thanks

**Pull request merged**